### DISCUSSION TITLE
Default SMTP settings - vulnerable?

### DISCUSSION THREAD

**Discussion by wildy** [globaleaks/globaleaks-whistleblowing-software#3650] [Q&A]
Hi people,

TL;DR: having a default SMTP server configured that can send mail to anybody on the internet is a bad thing.

I have just installed a test instance of GlobaLeaks, and noticed immediately that it had a pre-defined SMTP server, 'mail.globaleaks.org' configured for sending out notifications.

Of course, keeping a semi-open relay like this is incredibly bad practice and one can send lots of spam this way. How does Globaleaks make sure that no malicious mail gets through this relay? I'm assuming, some kind of tight filtering by mail content? Even so, this is not bulletproof.

**Comment by evilaliv3** [comment]
Hello @wildy 

Thank you for your question.

We could not enter into the specific details, but i can say that there are mixed strategies we use and recipes that enable the our SMTP server to serve our community for demo purposes and that try to limit abuses.

Do you see any alternative enabling users to test the technology right away after setup?

**Comment by wildy** [comment]
I don't have an alternative, though I appreciate your efforts to make it as simple as possible. Thank you for clarification.
